beancount/loader_test.py

```diff
@@ -479,8 +492,15 @@ def test_load_glob_relative_abs(self):
         self.assertEqual(3, len(entries))
         self.assertTrue(all(path.isabs(filename) for filename in options_map["include"]))
         self.assertEqual(
-            ["apples.beancount", "bananas.beancount", "oranges.beancount", "top.beancount"],
-            list(map(path.basename, options_map["include"])),
+            sorted(
+                [
+                    "apples.beancount",
+                    "bananas.beancount",
+                    "oranges.beancount",
+                    "top.beancount",
+                ]
+            ),
```

Comment:
It does not matter much, but sorting a literal list seems unnecessary: if the order matters, just write the literal in the correct order, if the order does not matter (and there aren't duplicates) use a set.